### PR TITLE
Sshfp support

### PR DESCRIPTION
**Pull request by dmykarp**
Still reading on the topic if there could be some unsupported combination of algorithm and digests. Will add tests if there are.

### CODE REVIEW

**Comment by dmykarp** [comment]
RFCs don't mention any restrictions on combinations. Any allowed value can be used with any other allowed value.

File: mreg/api/v1/zonefile.py [hunk 1]
```diff
@@ -98,6 +98,18 @@ def mx_zf_string(self, name, ttl, priority, mx):
...
 
+    def sshfp_zf_string(self, name, ttl, algorithm, hash_type, fingerprint):
+
+        data = {
+            'name': name,
+            'ttl': ttl,
+            'record_type': "SSHFP",
+            'algorithm': algorithm,
+            'hash_type': hash_type,
+            'fingerprint': f'{fingerprint}'
```
Comment (oyvindkolbu) [inline mreg/api/v1/zonefile.py]: Uneeded f-string.

File: mreg/models.py [hunk 2]
```diff
@@ -327,6 +327,20 @@ def loc_string(self, zone):
...
+    ttl = models.IntegerField(blank=True, null=True, validators=[validate_ttl])
+    algorithm = models.IntegerField(choices=((1, 'RSA'), (2, 'DSS'), (3, 'ECDSA'), (4, 'Ed25519')))
+    hash_type = models.IntegerField(choices=((1, 'SHA-1'), (2, 'SHA-256')))
+    fingerprint = models.CharField(max_length=64, validators=[validate_hexadecimal])
+
+    class Meta:
+        db_table = 'sshfp'
+
+    def __str__(self):
+        return str(self.host)
```
Comment (oyvindkolbu) [inline mreg/models.py]: Better __str__? Now all objects with the same host will be printed equally.

File: mreg/api/v1/zonefile.py (hunk 1 above)
Comment (dmykarp) [reply mreg/api/v1/zonefile.py]: You mean it can be just
'fingerpring': fingerprint ?

File: mreg/models.py (hunk 2 above)
Comment (dmykarp) [reply mreg/models.py]: Sorry, don't get it. the function name should be str?

File: mreg/api/v1/zonefile.py (hunk 1 above)
Comment (oyvindkolbu) [reply mreg/api/v1/zonefile.py]: Yes

File: mreg/models.py (hunk 2 above)
Comment (oyvindkolbu) [reply mreg/models.py]: Wrong usage of github-markup. Should say better `__str__`. Something like:
`return "f{self.host.name} -> {self.algorithm} ({self.get_algorithm_display()}) {self.hash_type} ({self.get_hash_type_display()}) {self.fingerprint}"`

Though wrap/make it shorter.

**Pull request merged**